testing/skia_gold_client/lib/skia_gold_client.dart

```diff
@@ -249,6 +249,8 @@ class SkiaGoldClient {
       '--work-dir', _tempPath,
       '--test-name', cleanTestName(testName),
       '--png-file', goldenFile.path,
+      // Otherwise post submit will not fail.
+      '--passfail',
```

Comment:
I don't think this will work. It does not throw for a non-zero exit code below. It just prints. 🙃